docs/guidelines.md

```diff
@@ -11,10 +11,8 @@ In the following paragraphs, keywords such as **MUST**, **SHOULD**, **COULD**
 are to be interpreted as in [RFC2119](https://www.ietf.org/rfc/rfc2119.txt);
 given that we recognise the following four output states and their restrictions:
 
-- `approve_as_optimal`: **MUST** be a best solution, **SHOULD** be without
-  comment
-- `approve_with_comment`: **COULD** be a best solution, **MUST** be with comment
-- `disapprove_with_comment`: **MUST** be with comment
+- `approve`: **MUST** be an approvable solution, **COULD** be without comment.
```

Comment:
I was wondering if we should also state this in the positive form: `**COULD** be with comment`? The positive form is usally a bit easier to read, although an argument against this is that most approval will probably have at least one comment (optimal approval being less likely).

Reply:
Agreed.